packages/dynamic/src/assignInlineVars.ts

```diff
@@ -0,0 +1,45 @@
+import {
+  walkObject,
+  get,
+  getVarName,
+  Contract,
+  MapLeafNodes,
+} from '@vanilla-extract/private';
+
+type Styles = { [cssVarName: string]: string };
+
+export function assignInlineVars(vars: Record<string, string>): Styles;
+export function assignInlineVars<ThemeContract extends Contract>(
+  contract: ThemeContract,
+  tokens: MapLeafNodes<ThemeContract, string>,
+): Styles;
+export function assignInlineVars(varsOrContract: any, tokens?: any) {
+  const styles: { [cssVarName: string]: string } = {};
```

Comment:
You could remove the repeated type.
```suggestion
  const styles: Styles = {};
```